code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -309,6 +309,38 @@
 	afterattack(obj/target, mob/user , flag)
 		return
 
+	get_desc(mob/user)
+		if(!usr.traitHolder?.hasTrait("training_chef"))
```

Comment:
```suggestion
		if(!user.traitHolder?.hasTrait("training_chef"))
```

Reply:
funnily enough user was the original idea, but using it spits out a runtime and stops it from working.

`in food_and_drink.dm, line 313: Cannot read 3.traitHolder`

I dont know why it does that :/

Reply:
Hm I think you might have get_desc arguments wrong. First is probably distance and the second is user.

Reply:
ah, oops :P thats on me for not understanding get_desc then